### PR TITLE
Add more prose to the readme

### PR DESCRIPTION
**Pull request by jcbhmr**
this pr would:
- just add some more text to the readme

<table><th>before <th>after
<tr><td>

image

<td>

image

</table>

feel free to edit this yourself to make the wording or sentences better 👍

### CODE REVIEW

**Comment by jcbhmr** [comment]
@mering i learned my lesson earlier; this is your feature and your merge/notmerge stuff. im just some random contributor 😃

File: README.md [hunk 1]
```diff
@@ -1,12 +1,35 @@
 # Bazel Dev Container Feature
 
-💚 Installs Bazelisk and Buildifier
+💚 Installs Bazel and some extra tools
```
Comment (mering) [inline README.md]: Feel free to decline this if you don't agree:

```suggestion
💚 Installs Bazel tooling
```

File: README.md [hunk 2]
```diff
@@ -21,7 +44,8 @@ Install [Bazelisk](https://github.com/bazelbuild/bazelisk) and [Buildifier](http
 
 ### Options
 
-You can specify a `bazelisk_version` and a `buildifier_version` to pin each of the tools like this:
+You can specify a `bazelisk_version` and a `buildifier_version` to pin each of
+the tools like this:
```
Comment (mering) [inline README.md:54]: As you are at it, can you please add the `v` before the version string (like in the default option) which I fixed in the tests earlier?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
+> - **Extensible to your needs**: Easily add support for new languages and
+>   platforms with Bazel's familiar extension language. Share and re-use
+>   language rules written by the growing Bazel community.
+
+<!-- prettier-ignore -->
+&mdash; [bazelbuild/bazel: a fast, scalable, multi-language and extensible build system](https://github.com/bazelbuild/bazel)
+
+This Feature installs Bazel using [Bazelisk]. Then, it symlinks
+`/usr/local/bin/bazel` to the Bazelisk installation so that you can use `bazel`
+as normal through `bazelisk`. It also installs the [Bazel VS Code extension].
```
Comment (mering) [inline README.md]: The main feature of Bazelisk is that it automatically uses the correct Bazel version based on your environment (file, env var, ...). Feel free to add it or not.

**Review by mering — COMMENTED** [review]
Thanks a lot for taking the initiative to improve the documentation! I left you a few comments. Consider that I am not a native speaker, so my suggestions might not always improve things. Feel free to object them in a reply.

**Review by mering — APPROVED** [review]
Awesome thanks. Approve, you can go ahead with merging!

**Pull request merged**